PopPUNK/models.py

```diff
@@ -505,32 +513,78 @@ def fit(self, X, max_num_clusters, min_cluster_prop):
 
         # DBSCAN parameters
         cache_out = "./" + self.outPrefix + "_cache"
-        min_samples = max(int(min_cluster_prop * self.subsampled_X.shape[0]), 10)
+        min_samples = min(max(int(min_cluster_prop * self.subsampled_X.shape[0]), 10),1023)
```

Comment:
This line is very confusing, can you split into two?

Reply:
Sorry, that's an appalling bit of code, simplified in 5454f7a.

Reply:
Oops, fixed in f721ca1.